L/libcgal_julia/build_tarballs.jl

```diff
@@ -42,15 +35,14 @@ case $target in
 esac
 """ * """
 ## configure build
-cmake $jlcgaldir -B /tmp/build """ * raw"""\
-  `# cmake specific` \
-  -DCMAKE_TOOLCHAIN_FILE=$CMAKE_TARGET_TOOLCHAIN \
+cmake $jlcgaldir """ * raw"""\
   -DCMAKE_BUILD_TYPE=Release \
-  -DCMAKE_FIND_ROOT_PATH=$prefix \
-  -DCMAKE_INSTALL_PREFIX=$prefix \
+  -DCMAKE_FIND_ROOT_PATH=${prefix} \
+  -DCMAKE_INSTALL_PREFIX=${prefix} \
+  -DCMAKE_TOOLCHAIN_FILE=$CMAKE_TARGET_TOOLCHAIN \
+  -DJulia_PREFIX=${prefix} \
```

Comment:
I took the liberty of sorting these arguments alphabetically and generally aligning them with other JLLs, in the hope that this will help a bit with future maintenance.